Topics/User_Experience/Mobile-First_Design.md

```diff
@@ -0,0 +1,58 @@
+# Mobile-First Design for Responsive Web Apps
+
+## What is it?
+
+Mobile-first design is an approach that prioritizes designing the UI of web apps for mobile devices first before other larger screens. 
+
+The reasoning behind this strategy is to promote _progressive enhancement_ — the UI’s mobile version will contain only the main functionalities required for the web app, and adding extra features that can be displayed on larger screens can come later. In contrast, using the method of desktop-first design and adding responsiveness after is called _graceful degradation_ because a lot of the details and complexities incorporated in the desktop version of the UI will have to be simplified and condensed to fit on a much smaller screen.
+
+## Benefits
+
+
+
+* Enhanced UX for mobile users
+    * As accessing web apps through phones has become the dominant mode of interaction — with more than [50%](https://www.statista.com/statistics/277125/share-of-website-traffic-coming-from-mobile-devices/#:~:text=Mobile%20accounts%20for%20approximately%20half,since%20the%20beginning%20of%202017.) of web traffic coming from mobile devices since 2022 — creating UIs with the mobile-first design approach ensures that the UX for mobile devices is not compromised from graceful degradation
+* Prioritized content
```

Comment:
Great point ! However, maybe you can expand on this point by including examples of what prioritized content will look like.  Additionally, maybe you can add some links to resources that show the differences between good prioritized content and bad prioritized content.

Reply:
Great idea! I'll be sure to make those changes